config/application.rb

```diff
@@ -30,14 +30,16 @@ class Application < Rails::Application
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
 
-    config.load_defaults 6.1
+    config.load_defaults 7.0
 
     config.log_tags = [:host, :uuid]
 
     config.cache_store = :memory_store
 
     config.active_job.queue_adapter = :delayed_job
 
+    config.active_support.cache_format_version = 7.0
```

Comment:
we don't need this since it's part of `load_defaults 7.0`https://guides.rubyonrails.org/configuring.html#config-active-support-cache-format-version